packages/backend/src/ee/services/AiAgentService.ts

```diff
@@ -3373,6 +3373,20 @@ export class AiAgentService {
         await this.aiAgentModel.deleteEval(evalUuid);
     }
 
+    async updateEvalRunResult(
+        evalRunUuid: string,
+        resultUuid: string,
+        error: Error | string,
+    ) {
+        await this.aiAgentModel.updateEvalRunResult(resultUuid, {
+            status: 'failed',
```

Comment:
do we want this function to always update to failures? Should we  use a more descriptive name then? maybe something like `recordEvalRunResultFailure` ?